tests/skipCrossGenFiles.arm.txt

```diff
@@ -0,0 +1,2 @@
+mscorlib.dll
```

Comment:
This had been explicitly skipped in earlier code.  I do not believe this has to be skipped, but since it is built by build.sh, skipping seems reasonable.

System.Private.CoreLib.dll could be added for similar reasons.